tajo-common/src/main/java/org/apache/tajo/exception/ErrorMessages.java

```diff
@@ -72,6 +72,7 @@
     ADD_MESSAGE(UNDEFINED_INDEX_FOR_TABLE, "index ''%s' does not exist", 1);
     ADD_MESSAGE(UNDEFINED_INDEX_FOR_COLUMNS, "index does not exist for '%s' columns of '%s' table", 2);
     ADD_MESSAGE(UNDEFINED_INDEX_NAME, "index name '%s' does not exist", 1);
+    ADD_MESSAGE(UNDEFINED_TABLESPACE_HANDLER, "No tablespace handler for URI scheme '%s'", 1);
```

Comment:
Almost all error messages of ours show just the fact. One example is ``invalid Session '%s'``, and another example is ``index name '%s' does not exist``. Users cannot know what they mean actually if they do not have knowledge about internal. This is because ``index name`` are likely to be generated automatically and many users do not know what is the session.

Our case is the same. Its fact is that there is no tablespace handler to support the uri scheme. Your suggested error message ``Please check your table location uri scheme`` is actually based on some assumption that users may mistypo the uri. But, it may not be true in some cases. Also, this error message is inconsistent with others.